packages/cursorless-vscode-e2e/src/suite/fixtures/recorded/languages/typescript/changeType5.yml

```diff
@@ -0,0 +1,23 @@
+languageId: typescript
+command:
+  version: 6
+  spokenForm: change type
+  action:
+    name: clearAndSetSelection
+    target:
+      type: primitive
+      modifiers:
+        - type: containingScope
+          scopeType: {type: type}
+  usePrePhraseSnapshot: true
+initialState:
+  documentContents: const value = <number>2;
+  selections:
+    - anchor: {line: 0, character: 15}
+      active: {line: 0, character: 15}
+  marks: {}
+finalState:
+  documentContents: const value = <>2;
```

Comment:
I'd argue removal range should include the `<>`, but not sure it's worth the effort using old tooling. Hesitant to introduce these new semantics half-working because we don't want to invest in upgrading the scope 🤔

Reply:
Ok but please file an issue to fix the removal range here; I don't love introducing new semantics that half-work

Reply:
Filed https://github.com/cursorless-dev/cursorless/issues/1892